src/main/scala/io/picnicml/doddlemodel/preprocessing/RangeScaler.scala

```diff
@@ -0,0 +1,52 @@
+package io.picnicml.doddlemodel.preprocessing
+
+import breeze.linalg.{*, Axis, max, min}
```

Comment:
Please use `Code -> Optimize Imports` if using IntelliJ (not sure what the command is in other IDEs) to reorder imports lexicographically.